package.json

```
@@ -49,9 +49,10 @@
         "bootstrap": "4.3.1",
         "countrily-data": "^1.0.1",
         "country-region-data": "^1.4.4",
+        "d3-shape": "^1.3.5",
         "emotion-theming": "^10.0.10",
         "eslint-plugin-import": "^2.17.3",
-        "graphql": "^14.3.1",
+        "graphql": "^14.4.2",
```

This seemed to fix the errors you were getting adding apollo.

`yarn gql:schema` needs to be run for the default mocks to work I believe. I don't know if you had that schema.json downloaded when you were testing. Actually I guess it should throw an error on import so nvm but good to keep it up to date...

---

Yes, looks like it solves the issue... Yeah, I had the schema.json downloaded during testing.